superset/datasets/api.py

```diff
@@ -332,7 +332,6 @@ def post(self) -> Response:
 
     @expose("/<pk>", methods=("PUT",))
     @protect()
-    @safe
     @statsd_metrics
     @event_logger.log_this_with_context(
         action=lambda self, *args, **kwargs: f"{self.__class__.__name__}.put",
```

Comment:
Can we write a small test verifying the payload comes back as SIP-40 compliant?

Reply:
Done